### ISSUE TITLE
Improve Developer Experience for the library. 

### ISSUE THREAD

**Issue by bidzyyys** [OpenZeppelin/rust-contracts-stylus#368]
### What is the feature you would like to see?

The goal of this issue to make usage of the library more developer friendly. This task includes:
- [ ] Improving documentation;
- [ ] Perfect scenario is that anyone can use the template for Stylus smart contract and use our library by just adding them to `Cargo.toml`, like:
    ```toml
        [dependencies]
        openzeppelin-stylus = "=v0.1.0"
        openzeppelin-crypto = "=0.1.0"
        
        [dev-dependencies]
        motsu = "=v0.1.0"
   ```

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines

**Comment by ggonzalez94** [comment]
@bidzyyys  can we mark this as closed after #373 and release 0.1.1? Or was there anything else you had in mind?

**Comment by bidzyyys** [comment]
Nope, we can close it with #373
